src/views/footer.html

```diff
@@ -9,7 +9,7 @@
     <a rel="next" href="/faqs">FAQs</a>
   </li>
   <li>
-    <a rel="next" href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH Code of Conduct</a>
+    <a rel="external" href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH Code of Conduct</a>
   </li>
 </ul>
-<small>&copy; Hack Cambridge 2018. All rights reserved.</small>
+<small>&copy; Hack Cambridge {{ now.getFullYear() }}. All rights reserved.</small>
```

Comment:
Suggest we use `{{ now.getFullYear() }} Hack Cambridge` to avoid it looking like "Hack Cambridge 2018" is the name of the organisation.